yarn.lock

```diff
@@ -0,0 +1,12568 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
I setup in the readme to not use yarn but npm. With the higher npm and our current node, we can stick to it as we have the package-lock.json ( I think it would be easier )

Reply:
Got it. I normally use yarn because npm is slow and sometimes not working. And node-gyp ugly errors,